client/src/components/License/LicenseSelector.vue

```diff
@@ -105,6 +105,8 @@ watchImmediate(
             track-by="licenseId"
             :options="licenseOptions"
             label="name"
+            selectLabel=""
+            deselectLabel=""
```

Comment:
```suggestion
            select-label=""
            deselect-label=""
```

To prevent
`Attribute 'selectLabel' must be hyphenated.`
eslint [vue/attribute-hyphenation](https://eslint.vuejs.org/rules/attribute-hyphenation.html)

Reply:
Thanks!